test/bondingEvent.js

```diff
@@ -268,15 +268,15 @@ describe('BondingEvent', async () => {
     });
 
     describe('excess usdt', async () => {
```

Comment:
relates to my question above when switching the other token for seuro.
rename if it does not deal with excess usdt.